README.rst

```diff
@@ -1,51 +1,47 @@
-but it should be later kept in a permanent storage.
-Circuits will be installed at the time of request or at a predefined time, and can also have a time to be deleted.
-The Napp also will listen for PortStatus events, modifying circuits that use a port that went down.
+Features
+========
+- REST API to create/modify/delete circuits;
+- REST API to create/modify/delete circuit scheduling;
+- list of circuits in memory and also synchronized to a permanent storage;
+- circuits can be installed at time of request or have an installation schedule;
+- circuits can use a predefined path or find a dynamic path;
+- the NApp will move circuits to another path in case of a link down;
+- web UI for circuits management.
 
-The cookie flow field is used to identify to which EVC the flow belongs. The first two bytes of the cookie is a prefix identifying the NApp using it,
-and the remaining 14 bytes are the EVC id.
 
 Requirements
 ============
 - kytos/flow_manager
 - kytos/pathfinder
 - kytos/topology
+- amlight/sndtrace_cp
 
 
 .. TAGs
 
-.. |Experimental| image:: https://img.shields.io/badge/stability-experimental-orange.svg
-   :target: https://github.com/kytos/mef_eline
-.. |License| image:: https://img.shields.io/github/license/kytos/kytos.svg
-   :target: https://github.com/kytos/mef_eline/blob/master/LICENSE
-.. |Build| image:: https://scrutinizer-ci.com/g/kytos/mef_eline/badges/build.png?b=master
+.. |Stable| image:: https://img.shields.io/badge/stability-stable-green.svg
+   :target: https://github.com/kytos-ng/mef_eline
+.. |License| image:: https://img.shields.io/github/license/kytos-ng/kytos.svg
+   :target: https://github.com/kytos-ng/mef_eline/blob/master/LICENSE
+.. |Build| image:: https://scrutinizer-ci.com/g/kytos-ng/mef_eline/badges/build.png?b=master
    :alt: Build status
-   :target: https://scrutinizer-ci.com/g/kytos/kytos/?branch=master
-.. |Coverage| image:: https://scrutinizer-ci.com/g/kytos/mef_eline/badges/coverage.png?b=master
+   :target: https://scrutinizer-ci.com/g/kytos-ng/kytos/?branch=master
+.. |Coverage| image:: https://scrutinizer-ci.com/g/kytos-ng/mef_eline/badges/coverage.png?b=master
    :alt: Code coverage
-   :target: https://scrutinizer-ci.com/g/kytos/mef_eline/
-.. |Quality| image:: https://scrutinizer-ci.com/g/kytos/mef_eline/badges/quality-score.png?b=master
+   :target: https://scrutinizer-ci.com/g/kytos-ng/mef_eline/
+.. |Quality| image:: https://scrutinizer-ci.com/g/kytos-ng/mef_eline/badges/quality-score.png?b=master
    :alt: Code-quality score
-   :target: https://scrutinizer-ci.com/g/kytos/mef_eline/
\ No newline at end of file
+   :target: https://scrutinizer-ci.com/g/kytos-ng/mef_eline/
+.. |Tag| image:: https://img.shields.io/github/tag/kytos-ng/mef_eline.svg
+   :target: https://github.com/kytos-ng/mef_eline/tags
+   :target: https://github.com/kytos-ng/mef_eline
```

Comment:
@ajoaoff oh, apparently it duplicated the `target` here for the `Tag`, not sure if it'll pick it up correctly, maybe be needed to delete line 47

Reply:
Done